### ISSUE TITLE
[FEAT]: make subs the same size 

### ISSUE THREAD

**Issue by No-Dot7369** [nova-video-player/aos-AVP#1311]
### Description

Subs when watching a 4k video are very small compared to normal size when watching 1080. Is there any way to keep them the same size when watching 1080 and 4k without having to make the size bigger every time I watch a 4k compared to watching 1080?

### Additional information

Nvidia Shield Pro with external HDD plugged into it using the latest Nova Video Player

**Comment by courville** [comment]
If this is a pgs internal sub and you are using latest nova github release, I think I have corrected the scaling this afternoon (pgs is work in progress in current github release).

**Comment by No-Dot7369** [comment]
Yes it is pgs the latest I see on GitHub which I am using is 6.3.4. Do you
mean you have corrected it for the next update?

**Comment by i990049** [comment]
Is it possible to adjust the size of the subtitles by percentage and the position of the subtitles can be moved up and down through remote control？dangbei player can do this.
https://www.dangbei.com/player/
